platform/ui/src/components/ButtonGroup/ButtonGroup.mdx

```diff
@@ -0,0 +1,120 @@
+---
+name: ButtonGroup
+menu: Components
+route: components/buttonGroup
+---
+
+import { Playground, Props } from 'docz';
+import Button from '../Button';
+import IconButton from '../IconButton';
+import ButtonGroup from './';
+
+# ButtonGroup
+
+Group Button display buttons together sharing its configurations
+
+## Import
+
+```javascript
+import { ButtonGroup } from '@ohfi/ui';
+```
+
+## Basic button group
+
+<Playground>
+  <div className="flex flex-col items-center">
+    <ButtonGroup color="primary" className="m-2">
+      <Button>One</Button>
+      <Button>Two</Button>
+      <Button>Tree</Button>
+    </ButtonGroup>
+    <ButtonGroup color="primary" variant="contained" className="m-2">
+      <Button>One</Button>
+      <Button>Two</Button>
+      <Button>Tree</Button>
+    </ButtonGroup>
+    <ButtonGroup color="primary" variant="text" className="m-2">
+      <Button>One</Button>
+      <Button>Two</Button>
+      <Button>Tree</Button>
+    </ButtonGroup>
+  </div>
+</Playground>
+
+## Group sizes and colorss
+
+<Playground>
+  <div className="flex flex-col items-center">
+    <ButtonGroup color="default" size="small" className="m-2">
+      <Button>One</Button>
+      <Button>Two</Button>
+      <Button>Tree</Button>
+    </ButtonGroup>
+    <ButtonGroup color="primary" size="medium" className="m-2">
+      <Button>One</Button>
+      <Button>Two</Button>
+      <Button>Tree</Button>
+    </ButtonGroup>
+    <ButtonGroup color="secondary" size="large" className="m-2">
+      <Button>One</Button>
+      <Button>Two</Button>
+      <Button>Tree</Button>
+    </ButtonGroup>
+  </div>
+</Playground>
+
+## Group orientation
+
+<Playground>
+  <div className="flex flex-row justify-center">
+    <ButtonGroup orientation="vertical" color="primary" className="m-2">
+      <Button>One</Button>
+      <Button>Two</Button>
+      <Button>Tree</Button>
+    </ButtonGroup>
+    <ButtonGroup
+      orientation="vertical"
+      color="primary"
+      variant="contained"
+      className="m-2"
+    >
+      <Button>One</Button>
+      <Button>Two</Button>
+      <Button>Tree</Button>
+    </ButtonGroup>
+    <ButtonGroup
+      orientation="vertical"
+      color="primary"
+      variant="text"
+      className="m-2"
+    >
+      <Button>One</Button>
+      <Button>Two</Button>
+      <Button>Tree</Button>
+    </ButtonGroup>
+  </div>
+</Playground>
+
+## Split Button
+
+<Playground>
+  <div className="flex flex-col items-center">
+    <ButtonGroup color="primary" className="m-2">
+      <Button>Search</Button>
+      <IconButton className="mr-2">
+        <svg
```

Comment:
This is just a demo page, but we should probably have an `<Icon />` component instead of in-lining SVGs by hand/

Reply:
Yes sir, that would be the ideal, as we don't have the `Icon` component yet, I've done this way. 
We can talk about the `Icon` component offline, and I can build it and update the docz